src/geo/transform.js

```diff
@@ -47,12 +48,13 @@ class Transform {
 
     constructor(minZoom: ?number, maxZoom: ?number, renderWorldCopies: boolean | void) {
         this.tileSize = 512; // constant
+        this.maxValidLatitude = 85.051129; // constant
 
         this._renderWorldCopies = renderWorldCopies === undefined ? true : renderWorldCopies;
         this._minZoom = minZoom || 0;
         this._maxZoom = maxZoom || 22;
 
-        this.setMaxBounds();
+        this.latRange = [this.maxValidLatitude * -1, this.maxValidLatitude];
```

Comment:
Let's move this back to `setMaxBounds` to make sure the behavior is consistent (and `setMaxBounds(null)` resets the map to the state it was by default).

Also nit: `-this.maxValidLatitude` is more readable than `this.maxValidLatitude * -1`.